src/auth/domain/user.domain.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString, Length } from 'class-validator';
+
+export class User {
```

Comment:
`User` and `RefreshToken` are DTOs, should be in the corresponding subfolder.

Reply:
done